src/components/decks/DeckListing.vue

```diff
@@ -5,12 +5,38 @@
       placeholder="Filter by title..."
       v-model:search="filterText"
       :is-disabled="isDisabled"></clearable-search>
+    <div
+      v-if="screenWidth < 768"
+      class="flex">
+      <phoenixborn-picker
+        class="flex-auto h-10 mb-4 pr-2 md:mb-0"
+        placeholder="Filter by Phoenixborn..."
+        v-model:filter="phoenixborn"
+        :is-legacy="shouldShowLegacyDecks"
+      />
+      <button
+        :class="[shouldShowLegacyDecks ? 'btn active h-10' : 'btn h-10']"
+        :title="[shouldShowLegacyDecks ? 'Show Reborn decks' : 'Show Legacy decks']"
+        @click="shouldShowLegacyDecks = !shouldShowLegacyDecks"
+      >
+        <i class="fas fa-archive"></i>
+      </button>
+    </div>
     <phoenixborn-picker
-      class="flex-auto h-10 mb-4 md:mb-0"
+      v-if="screenWidth >= 768"
+      class="flex-auto h-10 mb-4 md:pr-4 md:mb-0"
       placeholder="Filter by Phoenixborn..."
       v-model:filter="phoenixborn"
-      :is-legacy="showLegacy"
+      :is-legacy="shouldShowLegacyDecks"
     />
+    <button
+      v-if="screenWidth >= 768"
+      :class="[shouldShowLegacyDecks ? 'btn active h-10' : 'btn']"
+      :title="[shouldShowLegacyDecks ? 'Show Reborn decks' : 'Show Legacy decks']"
+      @click="shouldShowLegacyDecks = !shouldShowLegacyDecks"
+    >
+      <i class="fas fa-archive"></i>
```

Comment:
This pattern didn't exist when you started working on this branch, but as part of my efforts to increase the accessibility of the site, please add some text for screen readers using the `alt-text` style:

```suggestion
      <i class="fas fa-archive"></i>
      <span class="alt-text">{{ shouldShowLegacyDecks ? 'Show Reborn decks (showing legacy)' : 'Show Legacy decks (showing Reborn)' }}</span>
```